### PR TITLE
docs: Add SBOM page to operator guide

### PR DESCRIPTION
**Pull request by ericgregory**
Add documentation for generating and scanning SBOMs.

### CODE REVIEW

File: docs/deployment/security/sbom.mdx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: 'Software Bill of Materials (SBOM)'
+description: 'Generating SBOMs with wasmCloud'
+sidebar_position: 5
+---
+# Securing your wasmCloud supply chain with SBOMs
+
+[**Software Bills of Materials (SBOMs)**](https://edu.chainguard.dev/open-source/sbom/what-is-an-sbom/) describe the set of software dependencies (sometimes called the "software supply chain") used to build an application.
```
Comment (joonas) [inline docs/deployment/security/sbom.mdx]: I realized after I wrote up my notes that I should've really referred to the official source for this
```suggestion
[**Software Bills of Materials (SBOMs)**](https://www.cisa.gov/sbom) describe the set of software dependencies (sometimes called the "software supply chain") used to build an application. 
```

File: docs/deployment/security/sbom.mdx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+WebAssembly modules and components have the intrinsic advantage of bringing nothing else except for the direct dependencies needed for building the application itself, and generating an SBOM for a wasmCloud application is a straightforward process using standard tooling from the cloud native ecosystem.
+
+This document outlines the tooling and steps necessary to build and verify SBOMs for wasmCloud applications.
+
+### Prerequisites
+
+In addition to the wasmCloud tooling (`wash`) and language-specific tooling described in the [wasmCloud Quickstart](/docs/tour/hello-world?lang=tinygo), you will need the following tools:
+
+* [`syft`](https://github.com/anchore/syft?tab=readme-ov-file#installation) for SBOM generation
+* [`grype`](https://github.com/anchore/grype?tab=readme-ov-file#installation) (and optionally [`trivy`](https://github.com/aquasecurity/trivy?tab=readme-ov-file#get-trivy)) for vulnerability scanning
```
Comment (joonas) [inline docs/deployment/security/sbom.mdx]:
```suggestion
* [`grype`](https://github.com/anchore/grype?tab=readme-ov-file#installation) for scanning the SBOM
```

**Comment by joonas** [comment]
I'd love to add more guidance around how to upload this alongside your wasmcloud components, but I think that can be done follow-up in the interest of iteration

**Review by joonas — APPROVED** [review]
I'd love to add more guidance around how to upload this alongside your wasmcloud components, but I think that can and should be done follow-up in the interest of iterative improvement towards the version we want. Done is better than perfect 👍

**Review by brooksmtownsend — APPROVED** [review]

**Pull request merged**